generate.sh

```diff
@@ -69,7 +69,7 @@ require_command form  0.6.0
 generate_pac() {
     cecho $CYAN "Running svd2rust..."
     rm -rf src
-    svd2rust -i "${1}.svd" 2> >(tee svd2rust-warnings.log >&2)
+    svd2rust --nightly -i "${1}.svd" 2> >(tee svd2rust-warnings.log >&2)
```

Comment:
This should probably be a aparameter to the `generate_pac` function.

Reply:
It will end up in a bit of overengineering, since then we also need to pass the argument to `generate_pac` and thus also add it to the array.

Reply:
Done.